src/app/main/component/ubs/components/ubs-personal-information/ubs-add-address-pop-up/ubs-add-address-pop-up.component.html

```diff
@@ -14,11 +14,10 @@
       formControlName="district"
       class="shadow-none form-control"
       matGoogleMapsAutocomplete
-      [country]="'ua'"
       [types]="['geocode']"
-      (onAutocompleteSelected)="onDistrictSelected($event)"
+      [country]="'ua'"
```

Comment:
move it to constant

Reply:
✅ Done
commit: 816e650812df473d991fd7eeb3834df39451ab82